docs/README.md

```diff
@@ -201,6 +211,28 @@ die-if-failed = ["db.toml"]
 * **`die-if-failed` = `["<service-name>"]`**: As soon as any of the services defined in this the array fails, this
   service will be terminated as well.
 
+### ResourceLimit section
+
+> [!NOTE]
+> This feature requires running Horust as the root user or with related cgroups permissions.
+
+```toml
+[resource]
+cpu = 0.5
+memory = "100 MiB"
+pids-max = 100
+```
+
+* **`cpu` = `float`**: The maximum CPUs that the service can use. Negative value
+  means no limit. If unset, the default value will be `-1`.
```

Comment:
why not using an optional here as well instead of treating -1 as a value with special meaning?